README.md

```diff
@@ -46,6 +46,12 @@ which omits many dependencies (everything in `test_requirements.txt`).
 However, this will disable many functionalities and cause `import kats` to log
 warnings. See `setup.py` for full details and options.
 
+Kats is on conda-forge, so you can use `conda` to install it.
+
+```bash
+conda install -c conda-forge kats
```

Comment:
tried using conda myself. the following sequence worked, since Kats require python 3.8 to work:

```bash
# create conda environment
conda create --name kats python=3.8
conda activate kats

# install Kats
conda install -c conda-forge kats

```